### PR TITLE
Remove duplicate template parameter.

### PR DESCRIPTION
**Pull request by phprus**
Fix for issue #3140.

### CODE REVIEW

File: include/fmt/chrono.h [hunk 1]
```diff
@@ -2052,16 +2052,17 @@ struct formatter<std::chrono::duration<Rep, Period>, Char> {
   }
 };
 
-template <typename Char, typename Duration>
-struct formatter<std::chrono::time_point<std::chrono::system_clock, Duration>,
+template <typename Rep, typename Period, typename Char>
```
Comment (vitaut) [inline include/fmt/chrono.h:2055]: Why change template parameters? I suggest keeping the old ones at least for consistency with utc formatter and change the `format` function instead.

**Review by vitaut — CHANGES_REQUESTED** [review]
Thanks for the PR.

File: include/fmt/chrono.h (hunk 1 above)
Comment (phprus) [reply include/fmt/chrono.h:2055]: We need ``Rep`` and ``Period`` types for the condition on lines 2070-2071.

Comment (vitaut) [reply include/fmt/chrono.h:2055]: Yeah but these are available as `Duration::rep` and `Duration::period`. I don't think we need them here.

Comment (phprus) [reply include/fmt/chrono.h]: Done

**Review by vitaut — APPROVED** [review]

**Comment by vitaut** [comment]
Merged, thanks!

**Pull request merged**